includes/class-algolia-cli.php

```diff
@@ -107,10 +111,11 @@ public function reindex( $args, $assoc_args ) {
 	 *
 	 * @param Algolia_Index $index Algolia_Index instance.
 	 * @param bool          $clear Clear all existing records prior to pushing the records.
+	 * @param int           $from_page The page to start indexing from.
```

Comment:
Need to update `$from_page` to `$from_batch`

Reply:
@moory-se Gotcha! If you _want_ to setup a local WordPress instance, this is the most simple tool that can help you achieve that: https://localwp.com/